apps/src/bin/anoma-node/main.rs

```diff
@@ -11,7 +11,7 @@ fn main() -> Result<()> {
     color_eyre::install()?;
 
     // init logging
-    let default_directive = Directive::from_str("anoma=info")?;
+    let default_directive = Directive::from_str("namada=info")?;
```

Comment:
alternatively we could just do

```rust
    logging::init_from_env_or(LevelFilter::INFO)?;
```

like `namadac` and `namada` do

Reply:
:+1: have changed this to be `logging::init_from_env_or(LevelFilter::INFO)?` as well